parm/config/gfs/config.resources

```diff
@@ -615,7 +615,7 @@ case ${step} in
         exit 4
     esac
 
-    walltime="00:30:00"
+    walltime="02:00:00"
```

Comment:
If this is a case sensitive issue, we should put a `case $OCNRES` block around this so that the low resolution tests do not have to wait for such a big resources to come available.

Reply:
I wonder how much this is case sensitive versus # of ensemble members sensitive.

Reply:
reverting this because we are not using this job right now.